### ISSUE TITLE
Make button click create sample markdown when no text is selected

### ISSUE THREAD

**Issue by nomad5000** [jrm2k6/react-markdown-editor#36]
Reproduce:

I want to create a list:

click the list button without text selected:

I should get some sample markdown. Instead getting nothing as if the button doesn't work

**Comment by jrm2k6** [comment]
Yes, right now, markdown will be applied only if there is a selection. I will work on adding that feature, that shouldn't be really hard.
